### PR TITLE
Feature ejs relative includes

### LINKED ISSUE #103: Add support for relative EJS includes
EJS has two types of includes:

- *Relative includes*: includes resolved from the current template being rendered.
- *Root includes*: includes resolved from the root of the project, or from a specified includes root _(in eleventy, this is `_includes/`)_

Currently Eleventy supports the latter, but not the former, with all relative includes being resolved from the root `_includes/` folder.

For example:

```
|- views/
|--- _includes/
|----- component.example
|--- component.example
|--- component.md
```

`component.md`:
```
---
title: Component
---
<!-- This is a relative require; should resolve `views/component.example` -->
<%- include("component.example"); -%>

<!-- This is a root require; should resolve `views/_includes/component.example` -->
<%- include("/component.example"); -%>
```

I've submitted a rough PR that outlines how to update the markdown and EJS engines to properly support both types, w/o hurting the Eleventy `_includes/` functionality.

This also doesn't effect the `layout` front matter field, so I think it works fairly well.

#102

### PR DESCRIPTION
**Pull request by chrisdmacrae**
Closes #103

Adds support for relative includes in EJS templates.

### CODE REVIEW

**Comment by chrisdmacrae** [comment]
Need to review why the markdown tests are failing -- they work for me locally

File: src/TemplateRender.js [hunk 1]
```diff
@@ -149,12 +149,19 @@ TemplateRender.prototype.render = async function(str, data) {
   return this.engine.render(str, data);
 };
 
-TemplateRender.prototype.getCompiledTemplate = async function(str) {
+TemplateRender.prototype.getCompiledTemplate = async function(str, inputPath) {
   // TODO refactor better, move into TemplateEngine logic
   if (this.engineName === "md") {
-    return this.engine.compile(str, this.parseMarkdownWith, !this.useMarkdown);
+    return this.engine.compile(
```
Comment (kleinfreund) [inline src/TemplateRender.js]: You’re calling `compile` assuming an incorrect signature. The signature is:

```js
compile(str, inputPath, preTemplateEngine, bypassMarkdown)
```

But you pass `inputPath` last.

Comment (chrisdmacrae) [reply src/TemplateRender.js]: Yup, that was it!

**Comment by zachleat** [comment]
Hey @chrisdmacrae, do you think you could add a test for this to https://github.com/11ty/eleventy/blob/master/test/TemplateRenderEJSTest.js#L17 ? Should have some root include examples in there you can build from

**Comment by zachleat** [comment]
Actually looks like there is some commented out tests here that might be worth salvaging https://github.com/11ty/eleventy/blob/master/test/TemplateRenderEJSTest.js#L53

**Comment by zachleat** [comment]
Just an FYI if this makes it in today it’ll go out with the small maintenance release I’m planning to do later tonight!

**Comment by chrisdmacrae** [comment]
@zachleat I'll see what I can do!

**Comment by chrisdmacrae** [comment]
@zachlet tests for relative includes added. I duplicated the existing tests and ensured that it was running relative includes from the template's context.

Working like a charm.

Cheers 🍻

**Comment by zachleat** [comment]
Thank you!

**Pull request merged**